code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -69,48 +81,45 @@
 	usr.set_machine(src)
 	if(!beaker) return
 
-	if (href_list["create_vaccine"])
+	if (href_list["symptom"])
+		if(beaker && beaker.reagents)
+			if(beaker.reagents.reagent_list.len)
+				var/datum/reagent/blood/BL = locate() in beaker.reagents.reagent_list
+				if(BL)
+					if(BL.data && BL.data["viruses"])
+						var/list/viruses = BL.data["viruses"]
+						for(var/datum/disease/advance/D in viruses)
+							D.AddSymptom(new_symptoms[text2num(href_list["symptom"])])
+		src.updateUsrDialog()
```

Comment:
remove the unnecessary src in these calls as well (same goes for further examples below)